### PR TITLE
Add a load test to tcp module

### PR DESCRIPTION
**Pull request by Bhashinee**
## Purpose
$subject.
Related issue: https://github.com/ballerina-platform/ballerina-standard-library/issues/2173
Load test done for HTTP echo server example.

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: .github/workflows/trigger-load-tests.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  workflow_dispatch:
+    inputs:
+      tests:
+        description: >
+          List of test names
+          This needs to be filled only if you want to run a specific set of tests. Example: [“foo”, “bar”]
+        required: false
+      clusterName:
+        description: 'Cluster name'
+        default: 'bal-perf-cluster-test'
```
Comment (dilanSachi) [inline .github/workflows/trigger-load-tests.yml:13]:
```suggestion
        default: 'tcp-perf-cluster-test'
```
Better to change the cluster name

**Comment by dilanSachi** [comment]
Need to add `create-summary-pr.yaml` workflow also

File: .github/workflows/trigger-load-tests.yml (hunk 1 above)
Comment (Bhashinee) [reply .github/workflows/trigger-load-tests.yml]: Fixed with e28bc2f

**Review by dilanSachi — APPROVED** [review]
LGTM

**Pull request merged**